### PR TITLE
New method SFAPIclient::createProformaFromOrder

### PR DESCRIPTION
**Pull request by sajushko**
Method to create proforma from order

### CODE REVIEW

File: SFAPIclient/SFAPIclient.php [hunk 1]
```diff
@@ -920,6 +920,39 @@ public function createRegularFromProforma($proforma_id)
...
+    {
+        if (empty($order_id)) {
+            $this->last_error = array(
+                'status' => 404,
+                'message' => 'Item not found',
+            );
+            return null;
+        }
+        
+        $order = $this->get('/invoices/regular.json/' . $order_id);
```
Comment (superfaktura) [inline SFAPIclient/SFAPIclient.php:940]: `$order = $this->get('/invoices/order.json/' . $order_id);`

File: SFAPIclient/SFAPIclient.php [hunk 2]
```diff
@@ -920,6 +920,39 @@ public function createRegularFromProforma($proforma_id)
...
+        
+        if (empty($order)) {
+            $this->last_error = array(
+                'status' => 404,
+                'message' => 'Item not found',
+            );
+            return null;
+        }
+        
+        $order->Invoice->type = "proforma";
```
Comment (superfaktura) [inline SFAPIclient/SFAPIclient.php:950]: `$order->Invoice->type = 'proforma';`

File: SFAPIclient/SFAPIclient.php [hunk 3]
```diff
@@ -920,6 +920,39 @@ public function createRegularFromProforma($proforma_id)
...
+
```
Comment (superfaktura) [inline SFAPIclient/SFAPIclient.php:951]: `$order->Invoice->parent_id = $order_id;`

**Review by superfaktura — CHANGES_REQUESTED** [review]

**Comment by sajushko** [comment]
Changes done

**Review by superfaktura — APPROVED** [review]

**Pull request merged**